### ISSUE TITLE
Clarivate icons missing / bugged?

### ISSUE THREAD

**Issue by TSGut** [jpswalsh/academicons#193]
The Clarivate icon that was recently added seems to be bugged or missing for me? It also does not render for me on https://jpswalsh.github.io/academicons/ so I am unsure if the issue is on my end.

**Comment by GilOliveira** [comment]
Yep, can confirm it not working on neither the website or the web font.
